react-function-components/src/index.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+const root = document.getElementById('root');
+
+function CustomButton(props) {
+  return <button> Click Me! </button>
+}
+
+ReactDOM.render(CustomButton(),root);
```

Comment:
In React you don't call your components directly. You should be using JSX syntax here. If you directly call it you are breaking some of the functionality of the component, most directly you will break how `props` works.

Example:

```JSX
<CustomButton />
```